app/services/rbac.rb

```diff
@@ -5,10 +5,46 @@
 # This service is meant to handle calls to the RBAC API
 class Rbac
   API_CLIENT = RBACApiClient::AccessApi.new
-  COMPLIANCE_FULL_ACCESS = 'compliance:*:*'
   APPS = 'compliance,inventory'
   ANY = '*'
 
+  INVENTORY_VIEWER = 'inventory:*:read'
+  COMPLIANCE_VIEWER = 'compliance:policy:read' # universal read permission accross all roles
+  COMPLIANCE_ADMIN = 'compliance:*:*'
+
+  POLICY_READ = 'compliance:policy:read'
+  POLICY_CREATE = 'compliance:policy:create'
+  POLICY_DELETE = 'compliance:policy:delete'
+  POLICY_WRITE = 'compliance:policy:write'
+  REPORT_EXPORT_READ = 'compliance:report_export:read'
+  REPORT_READ = 'compliance:report:read'
+  SYSTEM_READ = 'compliance:system:read'
+
+  # Rails Model       Compliance RBAC resource
+  #
+  #                   │ system │ report │ policy │ report_export │
+  # Benchmark         │        │        │   x    │               │
+  # BusinessObjective │        │        │   x    │               │
+  # OsMajorVersion    │        │        │   x    │               │
+  # Profile           │        │        │   x    │               │
+  # RuleResult        │        │   x    │        │               │
+  # Rule              │        │        │        │               │
+  # SupportedSsg      │        │        │   x    │               │
+  #
+  # GraphQL Type      Compliance RBAC resource
+  #
+  #                   │ system │ report │ policy │ report_export │
+  # BusinessObjective │        │        │   x    │               │
+  # OperatingSystem   │   x    │        │        │               │
+  # OsMajorVersion    │        │        │   x    │               │
+  # Profile           │        │        │   x    │               │
+  # Query             │        │        │   x    │               │
+  # Rule              │   x    │        │        │               │
+  # RuleIdentifier    │   x    │        │        │               │
+  # System            │   x    │        │        │               │
+  # Tag               │   x    │        │        │               │
+  # TestResult        │        │   x    │        │      x        │
```

Comment:
Let's move this to the shared google drive in a table please and link it into the epic and the issue so QE can test it easier.